src/App.vue

```diff
@@ -1,33 +1,29 @@
 <template>
   <div id="app">
     <img alt="Vue logo" src="./assets/logo.png">
-    <Pipeline :steps="steps" :domains-list="domainsList" @selectedPipeline="setSteps" />
+    <Pipeline :steps="steps" :domains-list="domainsList" @selectedPipeline="setSteps"/>
```

Comment:
Maybe we could set up [Prettier](https://github.com/prettier/prettier) on this project to enforce a standard code-style?
Or/also: set up pug, which is a very light and convenient way of wring templates